browser/brave_browser_features.h

```diff
@@ -23,6 +23,7 @@ BASE_DECLARE_FEATURE(kBraveDayZeroExperiment);
 #if BUILDFLAG(IS_ANDROID)
 BASE_DECLARE_FEATURE(kNewAndroidOnboarding);
 #endif  // BUILDFLAG(IS_ANDROID)
+BASE_DECLARE_FEATURE(kEmailAliases);
```

Comment:
This should go in it's own feature file for email_aliases

Reply:
done